common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3975,6 +3975,16 @@
     ],
     "sqlState" : "21000"
   },
+  "SCALAR_FUNCTION_NOT_COMPATIBLE" : {
+    "message" : [
+      "Cannot find a compatible ScalarFunction#produceResult"
```

Comment:
Can we suggest users how to fix the issue?

Reply:
Replaced with

```
"ScalarFunction <scalarFunc> not overrides method 'produceResult(InternalRow)' with custom implementation."
```